setup.py

```diff
@@ -151,7 +156,7 @@
         "minio==7.1.0",
         "mock==2.0.0",
         "moto<4",
-        "mypy>=0.931",
+        "mypy>=0.931, <=0.981",
```

Comment:
@danielin917 why is this necessary?

Reply:
I was seeing a lot of unrelated lint errors before from different files. But actually now after removing the cap lint passed so not sure what changed